.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -58,7 +58,7 @@
            - [ ] Approve the `Approval - Partner notification` approval stage.
       1. - [ ] `Approval - Release` stage
            - [ ] **SYNC POINT**: Wait for Microsoft build release.
-           - [ ] ⚠️ 8.0: Push the release tag to `dotnet/dotnet`
+           - [ ] ⚠️ 8.0: In case you decided to not create the GitHub release (you set the pipeline parameter to `skip`), push the release tag to `dotnet/dotnet`
```

Comment:
I lean towards removing this.  This is adding unnecessary complexity to the checklist.  If we did this for every optional piece of functionality, the checklist would be unwieldy.

Reply:
I kept there an instruction verifying whether the tag belongs to the right SHA.